duckduckgo/iOS
# Add e2e test for email protection deep-links

Task/Issue URL: https://app.asana.com/0/0/1205692227651013/f
Tech Design URL:
CC:

**Description**:



Add a simple e2e test to verify that the e-mail protection menu entry correctly navigates to the e-mail protection website. 

**Steps to test this PR**:
_GitHub_
1. Verify that CI is green

_Local_
3. Start a simulator.
4. Install the current version of the app. 
5. Run `maestro test ./.maestro/release_tests/emailprotection.yaml` and verify the test passes.




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
